src/client/web/WebRTC.ts

```diff
@@ -0,0 +1,114 @@
+import Peer from 'peerjs'
+import Network from '../services/Network'
+
+export default class WebRTC {
+  private myPeer: Peer
+  private peers = {}
+  private videoGrid = document.getElementById('video-grid')
```

Comment:
you can use `document.querySelector('.video-grid')` after changing these from id to class

Reply:
Gotcha. Done!